### DISCUSSION TITLE
Profiles Behringer X-Touch Compact / Pfixer Xtremist

### DISCUSSION THREAD

**Discussion by mysticalpics** [rsjaffe/MIDI2LR#758] [General]
Is anyone shareing their Profiles for the Behringer X-Touch Compact with motorized sliders?
Maybe simimlar to the pfixer setup?

**Comment by MarcksHouse** [comment]
Good evening,
Have a look at the discussion below
'To XTouch Compact users #745' from 21th september
you may find a zip to download in the response I've made
hoping it will help you
Say-me

**Reply by mysticalpics** [reply]
thx alot... 
i spent some hours now with your config...
without success

i think you did this for microsoft plattforms, but im working on a mac...

also the mapping is very strange...
when i start, the sliders are mapped with luminance??? 
for me a really strange setup to be honest.

i want to work with the pfixer mappings, which makes more sense for my workflow of developing pics
the basic developing functions are mapped for the sliders, because thats what mostly is needed to adjust.

seems i have to do the pfixer mappings from scratch, which i try to avoid...

also im not so familiar with midi2lr... so maybe i can profit later from your excel table...

**Reply by MarcksHouse** [reply]
I understand, and recognise that it needs some more explanations,
Yes I'm on windows 11. 

Operation is based on 7 main menus accessible with line 3:

Library, Basic, TSL et Couleur, b&w/Curve, Crop/Transform, and, mask,

for each menu, you have 2 levels,  A and B.
The commands are disposed mainly in the same idea for all menus
On down right you have the main commun commands, undo/redo, navigation, reset, view.. level A & B.
For begining, assure you reset application link with the button ‘Reset App’ above the touch ‘A’. (sometimes you have triggering, press A and B alternatively.)

Then, enlight level A and press the menu ‘Basic’, (Ligne3-2nd), you are now acting on the develop mode, acting mostly on luminance.
Press the level B and the sliders changes for saturation.
This config allows me to develop rapidely and affect notes, colors, labels to my photos.

If I want to go further in color I activate the third Menu TSL-Color / A: hue+ Sat, B: Hue + Lum
Or, I go to the grey menu to develop in black and white

And so on.

I don’t find the library mode very useful but it can help in quick editing

Its dificult to memorise all configuration, and need practice,

To better understand each menus you need to open programming files
with a text editor, otherwise you lose the structure and comments in charging them in MIDI2LR

![Capure Menus excel](https://github.com/rsjaffe/MIDI2LR/assets/144238913/ee6da747-d156-417e-8590-2f1765832110)

**Comment by mysticalpics** [comment]
will first try to use it... for this i mapped just my most needed stuff:

`  <setting channel="1" note="47" command_string="Key1"/>
  <setting channel="1" note="48" command_string="Key2"/>
  <setting channel="1" note="50" command_string="Next"/>
  <setting channel="1" note="49" command_string="Prev"/>
  <setting channel="1" note="53" command_string="LRCopy"/>
  <setting channel="1" note="54" command_string="LRPaste"/>
  <setting channel="1" note="52" command_string="ResetAll"/>
  <setting channel="1" note="51" command_string="Undo"/>
  <setting channel="1" note="14" command_string="WhiteBalanceAuto"/>
  <setting channel="1" note="12" command_string="AutoTone"/>
  <setting channel="1" controller="24" command_string="Temperature"/>
  <setting channel="1" controller="22" command_string="Tint"/>
  <setting channel="1" controller="1" command_string="Exposure"/>
  <setting channel="1" controller="2" command_string="Contrast"/>
  <setting channel="1" controller="5" command_string="Highlights"/>
  <setting channel="1" controller="4" command_string="Shadows"/>
  <setting channel="1" controller="6" command_string="Whites"/>
  <setting channel="1" controller="3" command_string="Blacks"/>
  <setting channel="1" controller="7" command_string="Clarity"/>
  <setting channel="1" controller="8" command_string="Dehaze"/>
  <setting channel="1" controller="20" command_string="Vibrance"/>
  <setting channel="1" controller="18" command_string="Saturation"/>
  <setting channel="1" note="10" command_string="ResetVibrance"/>
  <setting channel="1" note="8" command_string="ResetSaturation"/>
  <setting channel="1" controller="9" command_string="PostCropVignetteAmount"/>
  <setting channel="1" note="32" command_string="CropAll"/>`

**Reply by MarcksHouse** [reply]
going slowly, going further

**Comment by mysticalpics** [comment]
![IMG_3290](https://github.com/rsjaffe/MIDI2LR/assets/152011336/80ffac64-55df-4b74-8793-51e4c1e5f08b)

**Reply by MarcksHouse** [reply]
Interesting layer,
I would like have one similar at the begining..

**Comment by heyjeff** [comment]
@mysticalpics How's your progress, I have the same setup and I'm just getting started.
